gen/pb-go/flyteidl/service/auth.swagger.json

```diff
@@ -0,0 +1,191 @@
+        "description": "Retrieves OAuth2 authorization server metadata. This endpoint is anonymously accessible.",
+        "operationId": "OAuth2Metadata",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/serviceOAuth2MetadataResponse"
+            }
+          }
+        },
+        "tags": [
+          "AuthService"
+        ]
+      }
+    },
+    "/config/v1/flyte_client": {
+      "get": {
+        "summary": "Anonymously accessible. Retrieves the client information clients should use when initiating OAuth2 authorization\nrequests.",
+        "description": "Retrieves public flyte client info. This endpoint is anonymously accessible.",
+        "operationId": "FlyteClient",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/serviceFlyteClientResponse"
+            }
+          }
+        },
+        "tags": [
+          "AuthService"
+        ]
+      }
+    },
+    "/me": {
+      "get": {
+        "summary": "Retrieves user information about the currently logged in user.",
+        "description": "Retrieves user info",
+        "operationId": "UserInfo",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/serviceUserInfoResponse"
+            }
+          }
+        },
+        "tags": [
+          "AuthService"
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "serviceFlyteClientResponse": {
+      "type": "object",
+      "properties": {
+        "client_id": {
+          "type": "string",
+          "description": "client_id to use when initiating OAuth2 authorization requests."
+        },
+        "redirect_uri": {
+          "type": "string",
+          "description": "redirect uri to use when initiating OAuth2 authorization requests."
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "scopes to request when initiating OAuth2 authorization requests."
+        },
+        "authorization_metadata_key": {
+          "type": "string",
+          "description": "Authorization Header to use when passing Access Tokens to the server. If not provided, the client should use the\ndefault http `Authorization` header."
+        }
+      },
+      "description": "FlyteClientResponse encapsulates public information that flyte clients (CLIs... etc.) can use to authenticate users."
+    },
+    "serviceOAuth2MetadataResponse": {
+      "type": "object",
+      "properties": {
+        "issuer": {
+          "type": "string",
+          "description": "Defines the issuer string in all JWT tokens this server issues. The issuer can be admin itself or an external\nissuer."
```

Comment:
nit: can we remove \n . Is it due to column width restriction

Reply:
I just ran generate... these files are all generated